src/main/java/com/ksm/robolo/roboloapp/domain/ClientEntity.java

```diff
@@ -14,6 +14,10 @@
 	@GeneratedValue(strategy = GenerationType.AUTO)
 	private Long id;
 
+	@ManyToOne(targetEntity = UserEntity.class)
```

Comment:
Dlaczego tutaj dajemy many to one?

Reply:
Żeby każdy klient miał referencje do użytkownika, do którego należy. Robimy, żeby wszystkie tabele należały do użytkownika